### PR TITLE
Implement valtimer ultimate

### PR DESCRIPTION
**Pull request by agustinesco**
Test with this [Backend PR](https://github.com/lambdaclass/mirra_backend/pull/298)

Now when you pick valtimer and try to cast his ultimate you'll place an area that damage and pull enemies to his center

Todo list:
- [x] Implement a visual representation of the damage pools
- [ ] Adjust range and damage

### CODE REVIEW

**Comment by BertovDev** [comment]
Tested on android s23 working fine.

**Comment by BertovDev** [comment]
We should tweak the config to adjust the range and radius of the ability

**Review by Nico-Sanchez — CHANGES_REQUESTED** [review]
It doesn't seem to work accurately. The ultimate's pool object doesn't spawn where I'm aiming and the visual representation size is not the same as the one received from the backend.
This can be seen by a character getting pulled in to a pool that it's not colliding yet in the visuals.

**Comment by BertovDev** [comment]
> It doesn't seem to work accurately. The ultimate's pool object doesn't spawn where I'm aiming and the visual representation size is not the same as the one received from the backend. This can be seen by a character getting pulled in to a pool that it's not colliding yet in the visuals.

You are right! all adjust are done!

File: client/Assets/Scripts/Skills/Skill.cs [hunk 1]
```diff
@@ -314,22 +315,26 @@ public float GetSkillRadius()
...
+                radius = skillInfo.skillCircleRange;
                 break;
         }
         return radius;
     }
 
     public void SetSkillRadius(float radius)
     {
-        skillInfo.skillCircleRadius = radius;
+        skillInfo.skillCircleRange = radius;
```
Comment (Nico-Sanchez) [inline client/Assets/Scripts/Skills/Skill.cs:326]: Shouldn't we rename this? Looks awkward to have a variable called "range" and assign a "radius" to it.

Comment (BertovDev) [reply client/Assets/Scripts/Skills/Skill.cs]: Yess, nice catch

Comment (BertovDev) [reply client/Assets/Scripts/Skills/Skill.cs]: Also renamed another vars.

**Comment by BertovDev** [comment]
Draft till we decide what to do with the scaled textures inputs.

**Comment by BertovDev** [comment]
> Draft till we decide what to do with the scaled textures inputs.

We will continue with this textures for now. Implementing a new one in another iteration.

**Review by Nico-Sanchez — DISMISSED** [review]

**Review by AngieDutra — DISMISSED** [review]
Tested on Desktop and Android and it's looking awesome!

**Review by AngieDutra — COMMENTED** [review]
I'll say this prefab is changing on use and shouldn't
Captura de pantalla 2024-03-11 a las 16 48 20 p  m

**Review by AngieDutra — APPROVED** [review]
Tested with a match between two desktops and worked just fine ✨ No prefabs modified in the process

**Review by ncontinanza — APPROVED** [review]
LGTM!

**Review by Nico-Sanchez — APPROVED** [review]

**Pull request merged**